arrocke/gloss-translation
# feat: create and remove multi word glosses

## What has changed

* Routes to add and delete phrases
* UI to select words to link in a phrase, link and unlink words, show words linked to current word
* Keyboard shortcuts to select words, link and unlink



## Connected Issues

closes #377 



## QA Steps

1. Use the checkbox and Ctrl+Enter the select/deselect words
2. Use the link words button and Ctrl+L to link selected words
3. Use the unlink words button and Ctrl+U to unlink selected words
4. Confirm that linked words can't be selected to form new phrases
5. Confirm that linked words show background when one of the words in the phrase is highlighted



## Post-Deployment
